src/commands/utils/detectors.ts

```diff
@@ -97,7 +97,6 @@ export async function getDetectorInfo(
 }
 
 export function getPortalUrl(clusterdata: ARMResponse<unknown>) {
```

Comment:
This looks like just a formatting change, and I think it'll cause a merge conflict with #358 - shall we skip this change?